os/include/tinyara/log_dump/log_dump.h

```diff
@@ -31,8 +31,8 @@
 #define LOG_DUMP_STACKSIZE   16384		/* Log dump thread stack size */
 #define LOG_DUMP_PRIORITY    100		/* Log dump thread priority */
 
-#define LOGDUMP_SAVE_START	"1"
-#define LOGDUMP_SAVE_STOP	"2"
+#define LOGDUMP_SAVE_START	"a"
```

Comment:
why is this change done?

Reply:
earlier to start log dump we would write 1 to the fd and to stop we would write 2 to the fd. With the newer implementation, when we write 'a' or 'b' to the fd, its to start or stop the log dump and when we write any number to the fd, it sets the number of last n blocks to be fetched. To avoid confusion between log dump start or stop and setting the last n chunks to be fetched, I have made the change.